src/internet_identity/src/assets.rs

```diff
@@ -19,23 +19,35 @@ pub fn init_assets() {
 }
 
 // The <script> tag that loads the 'index.js'
-const JS_SETUP_SCRIPT: &str = "let s = document.createElement('script');s.type = 'module';s.src = '/index.js';document.head.appendChild(s);";
+pub const JS_SETUP_SCRIPT1: &str = "let s = document.createElement('script');s.type = 'module';s.src = '/index.js';document.head.appendChild(s);";
```

Comment:
Yeah this is a quick hack. We're changing a couple of things in the way the assets are served and this will go away very soon. Will basically read the `indexX.js` bit from the HTML and then inject that. Thanks for checking though!